### PR TITLE
Test fix.

### PR DESCRIPTION
**Pull request by labkey-sweta**
#### Rationale
Test fix for 
TNPRC_EHRTest.testAnimalHistoryReports
There appears to be an error in the report: SNP

#### Related Pull Requests
* 

#### Changes
*

### CODE REVIEW

**Review by labkey-tchad — CHANGES_REQUESTED** [review]
Looks like `doAndWaitForRepeatedPageSignal` isn't respecting the timeout (the error says that it failed after waiting for 10 seconds). This means that this increase wouldn't have helped and that there's a problem with the helper.

File: ehr/test/src/org/labkey/test/pages/ehr/ParticipantViewPage.java [hunk 1]
```diff
@@ -384,11 +384,10 @@ public void select()
             {
                 log("Selecting Report: " + getLabel());
                 scrollIntoView(_el);
-                doAndWaitForRepeatedPageSignal(_el::click, REPORT_TAB_SIGNAL, Duration.ofSeconds(2));
+                doAndWaitForRepeatedPageSignal(_el::click, REPORT_TAB_SIGNAL, Duration.ofSeconds(3));
```
Comment (labkey-tchad) [inline ehr/test/src/org/labkey/test/pages/ehr/ParticipantViewPage.java]: Actually that `Duration` parameter isn't the timeout, that's how long it waits for the page signal to stop updating. This needs to use the overload of this method that takes a custom timeout.
```suggestion
                doAndWaitForRepeatedPageSignal(_el::click, REPORT_TAB_SIGNAL, Duration.ofSeconds(2), longWait());
```

**Review by labkey-tchad — CHANGES_REQUESTED** [review]

File: ehr/test/src/org/labkey/test/pages/ehr/ParticipantViewPage.java (hunk 1 above)
Comment (labkey-sweta) [reply ehr/test/src/org/labkey/test/pages/ehr/ParticipantViewPage.java]: Thank you

**Review by labkey-tchad — APPROVED** [review]
Seems to have done the trick

**Pull request merged**